fix: Correct @import Syntax in CLAUDE.md Files

## Summary

Fix incorrect `@import` syntax in CLAUDE.md files to comply with official Claude Code documentation. Currently, relative paths lack the required `./` prefix.

## Background (Why)

According to official Claude Code documentation, the import syntax requires:

- **Relative paths**: Must start with `./` (e.g., `@./token-management.md`)
- **Absolute paths**: Use full path (e.g., `@~/.claude/settings.md`)
- **Home directory**: Use `@~/` prefix

### Current State (Incorrect)

`~/.claude/CLAUDE.md`:
```markdown
@token-management.md
@conversation-language.md
@git-identity.md
@commit-settings.md
```

`~/Sources/CLAUDE.md`:
```markdown
@claude-guidelines/environment.md
@claude-guidelines/workflow.md
```

### Expected State (Correct)

`~/.claude/CLAUDE.md`:
```markdown
@./token-management.md
@./conversation-language.md
@./git-identity.md
@./commit-settings.md
```

`~/Sources/CLAUDE.md`:
```markdown
@./claude-guidelines/environment.md
@./claude-guidelines/workflow.md
```

## Proposed Solution (What/How)

### 1. Update Global CLAUDE.md

```markdown
# Claude Code Global Configuration

## Core Settings
@./token-management.md
@./conversation-language.md
@./git-identity.md
@./commit-settings.md
```

### 2. Update Project CLAUDE.md

```markdown
# Universal Development Guidelines

## Core Guidelines
@./claude-guidelines/environment.md
@./claude-guidelines/workflow.md
@./claude-guidelines/problem-solving.md
...
```

### 3. Add Validation in Scripts

Add syntax validation to `verify.sh`:
```bash
# Validate @import syntax
validate_import_syntax() {
    local file="$1"
    if grep -E '^@[^./~]' "$file" | grep -v '^@https'; then
        echo "[ERROR] Invalid import syntax in $file"
        echo "  Use @./path for relative or @~/path for home directory"
        return 1
    fi
}
```

## Acceptance Criteria

- [ ] Fix `global/CLAUDE.md` import syntax
- [ ] Fix `project/CLAUDE.md` import syntax
- [ ] Add import syntax validation to `verify.sh`
- [ ] Update all module files with correct syntax
- [ ] Test import resolution in Claude Code session

## Impact

- **Reliability**: Ensure imports are correctly resolved
- **Compatibility**: Full compliance with official documentation
- **Consistency**: Uniform syntax across all configuration files

## Priority

**Critical** - Current syntax may cause import failures

## References

- [Claude Code Memory Documentation](https://code.claude.com/docs/en/memory)
- Import syntax: `@./path` for relative, `@~/path` for home directory

--- a/global/CLAUDE.md
+++ b/global/CLAUDE.md
@@ -4,10 +4,10 @@ Global settings for all Claude Code sessions. Project-specific `CLAUDE.md` files
 
 ## Core Settings (Import Syntax)
 
-@token-management.md
-@conversation-language.md
-@git-identity.md
-@commit-settings.md
+@./token-management.md
+@./conversation-language.md
+@./git-identity.md
+@./commit-settings.md
 
 ## Priority Rules
 

--- a/plugin/skills/api-design/SKILL.md
+++ b/plugin/skills/api-design/SKILL.md
@@ -16,14 +16,14 @@ description: Provides API design guidelines for REST, GraphQL, versioning, loggi
 ## Reference Documents (Import Syntax)
 
 ### API Design
-@reference/api-design.md
+@./reference/api-design.md
 
 ### Architecture
-@reference/architecture.md
+@./reference/architecture.md
 
 ### Observability
-@reference/logging.md
-@reference/observability.md
+@./reference/logging.md
+@./reference/observability.md
 
 ## Core Principles
 

--- a/plugin/skills/coding-guidelines/SKILL.md
+++ b/plugin/skills/coding-guidelines/SKILL.md
@@ -15,16 +15,16 @@ description: Provides comprehensive coding standards for quality, naming convent
 ## Reference Documents (Import Syntax)
 
 ### Core Standards
-@reference/general.md
-@reference/quality.md
+@./reference/general.md
+@./reference/quality.md
 
 ### Error & Safety
-@reference/error-handling.md
-@reference/memory.md
+@./reference/error-handling.md
+@./reference/memory.md
 
 ### Performance
-@reference/concurrency.md
-@reference/performance.md
+@./reference/concurrency.md
+@./reference/performance.md
 
 ## Core Principles
 

--- a/plugin/skills/documentation/SKILL.md
+++ b/plugin/skills/documentation/SKILL.md
@@ -17,13 +17,13 @@ description: Provides documentation standards for README, API docs, code comment
 ## Reference Documents (Import Syntax)
 
 ### Documentation Standards
-@reference/documentation.md
+@./reference/documentation.md
 
 ### Language Conventions
-@reference/communication.md
+@./reference/communication.md
 
 ### Project Cleanup
-@reference/cleanup.md
+@./reference/cleanup.md
 
 ## Core Principles
 

--- a/plugin/skills/performance-review/SKILL.md
+++ b/plugin/skills/performance-review/SKILL.md
@@ -46,10 +46,10 @@ Profiling → Identify bottlenecks → Optimize → Verify
 - [ ] Cache hit rate monitoring
 
 ## Reference Documents (Import Syntax)
-@reference/performance.md
-@reference/memory.md
-@reference/concurrency.md
-@reference/monitoring.md
+@./reference/performance.md
+@./reference/memory.md
+@./reference/concurrency.md
+@./reference/monitoring.md
 
 ## Caution
 

--- a/plugin/skills/project-workflow/SKILL.md
+++ b/plugin/skills/project-workflow/SKILL.md
@@ -16,20 +16,20 @@ description: Provides workflow guidelines for problem-solving, git commits, GitH
 ## Reference Documents (Import Syntax)
 
 ### Workflow
-@reference/workflow.md
-@reference/question-handling.md
-@reference/workflow-problem-solving.md
-@reference/performance-analysis.md
+@./reference/workflow.md
+@./reference/question-handling.md
+@./reference/workflow-problem-solving.md
+@./reference/performance-analysis.md
 
 ### GitHub
-@reference/github-issue-5w1h.md
-@reference/github-pr-5w1h.md
-@reference/git-commit-format.md
+@./reference/github-issue-5w1h.md
+@./reference/github-pr-5w1h.md
+@./reference/git-commit-format.md
 
 ### Project Management
-@reference/problem-solving.md
-@reference/build.md
-@reference/testing.md
+@./reference/problem-solving.md
+@./reference/build.md
+@./reference/testing.md
 
 ## Core Principles
 

--- a/plugin/skills/security-audit/SKILL.md
+++ b/plugin/skills/security-audit/SKILL.md
@@ -35,9 +35,9 @@ allowed-tools: Read, Grep, Glob
 - [ ] Resource access control
 
 ## Reference Documents (Import Syntax)
-@reference/security.md
-@reference/error-handling.md
-@reference/api-design.md
+@./reference/security.md
+@./reference/error-handling.md
+@./reference/api-design.md
 
 ## OWASP Top 10 Reference
 

--- a/project/.claude/skills/api-design/SKILL.md
+++ b/project/.claude/skills/api-design/SKILL.md
@@ -24,14 +24,14 @@ model: sonnet
 ## Reference Documents (Import Syntax)
 
 ### API Design
-@reference/api-design.md
+@./reference/api-design.md
 
 ### Architecture
-@reference/architecture.md
+@./reference/architecture.md
 
 ### Observability
-@reference/logging.md
-@reference/observability.md
+@./reference/logging.md
+@./reference/observability.md
 
 ## Core Principles
 

--- a/project/.claude/skills/coding-guidelines/SKILL.md
+++ b/project/.claude/skills/coding-guidelines/SKILL.md
@@ -22,16 +22,16 @@ model: sonnet
 ## Reference Documents (Import Syntax)
 
 ### Core Standards
-@reference/general.md
-@reference/quality.md
+@./reference/general.md
+@./reference/quality.md
 
 ### Error & Safety
-@reference/error-handling.md
-@reference/memory.md
+@./reference/error-handling.md
+@./reference/memory.md
 
 ### Performance
-@reference/concurrency.md
-@reference/performance.md
+@./reference/concurrency.md
+@./reference/performance.md
 
 ## Core Principles
 

--- a/project/.claude/skills/documentation/SKILL.md
+++ b/project/.claude/skills/documentation/SKILL.md
@@ -23,13 +23,13 @@ model: sonnet
 ## Reference Documents (Import Syntax)
 
 ### Documentation Standards
-@reference/documentation.md
+@./reference/documentation.md
 
 ### Language Conventions
-@reference/communication.md
+@./reference/communication.md
 
 ### Project Cleanup
-@reference/cleanup.md
+@./reference/cleanup.md
 
 ## Core Principles
 

--- a/project/.claude/skills/performance-review/SKILL.md
+++ b/project/.claude/skills/performance-review/SKILL.md
@@ -53,10 +53,10 @@ Profiling → Identify bottlenecks → Optimize → Verify
 - [ ] Cache hit rate monitoring
 
 ## Reference Documents (Import Syntax)
-@reference/performance.md
-@reference/memory.md
-@reference/concurrency.md
-@reference/monitoring.md
+@./reference/performance.md
+@./reference/memory.md
+@./reference/concurrency.md
+@./reference/monitoring.md
 
 ## Caution
 

--- a/project/.claude/skills/project-workflow/SKILL.md
+++ b/project/.claude/skills/project-workflow/SKILL.md
@@ -24,20 +24,20 @@ model: sonnet
 ## Reference Documents (Import Syntax)
 
 ### Workflow
-@reference/workflow.md
-@reference/question-handling.md
-@reference/workflow-problem-solving.md
-@reference/performance-analysis.md
+@./reference/workflow.md
+@./reference/question-handling.md
+@./reference/workflow-problem-solving.md
+@./reference/performance-analysis.md
 
 ### GitHub
-@reference/github-issue-5w1h.md
-@reference/github-pr-5w1h.md
-@reference/git-commit-format.md
+@./reference/github-issue-5w1h.md
+@./reference/github-pr-5w1h.md
+@./reference/git-commit-format.md
 
 ### Project Management
-@reference/problem-solving.md
-@reference/build.md
-@reference/testing.md
+@./reference/problem-solving.md
+@./reference/build.md
+@./reference/testing.md
 
 ## Core Principles
 

--- a/project/.claude/skills/security-audit/SKILL.md
+++ b/project/.claude/skills/security-audit/SKILL.md
@@ -39,9 +39,9 @@ model: sonnet
 - [ ] Resource access control
 
 ## Reference Documents (Import Syntax)
-@reference/security.md
-@reference/error-handling.md
-@reference/api-design.md
+@./reference/security.md
+@./reference/error-handling.md
+@./reference/api-design.md
 
 ## OWASP Top 10 Reference
 

--- a/project/CLAUDE.md
+++ b/project/CLAUDE.md
@@ -7,43 +7,43 @@ Universal conventions for this repository. Works with global settings in `~/.cla
 **YOU MUST** check environment settings first for timezone and locale.
 
 ### Environment & Workflow
-@claude-guidelines/environment.md
-@claude-guidelines/workflow.md
-@claude-guidelines/problem-solving.md
-@claude-guidelines/communication.md
-@claude-guidelines/git-commit-format.md
-@claude-guidelines/common-commands.md
+@./claude-guidelines/environment.md
+@./claude-guidelines/workflow.md
+@./claude-guidelines/problem-solving.md
+@./claude-guidelines/communication.md
+@./claude-guidelines/git-commit-format.md
+@./claude-guidelines/common-commands.md
 
 ### Code Standards
-@claude-guidelines/coding-standards/general.md
-@claude-guidelines/coding-standards/quality.md
-@claude-guidelines/coding-standards/error-handling.md
-@claude-guidelines/operations/cleanup.md
+@./claude-guidelines/coding-standards/general.md
+@./claude-guidelines/coding-standards/quality.md
+@./claude-guidelines/coding-standards/error-handling.md
+@./claude-guidelines/operations/cleanup.md
 
 ### Technical
-@claude-guidelines/coding-standards/concurrency.md
-@claude-guidelines/coding-standards/memory.md
-@claude-guidelines/coding-standards/performance.md
+@./claude-guidelines/coding-standards/concurrency.md
+@./claude-guidelines/coding-standards/memory.md
+@./claude-guidelines/coding-standards/performance.md
 
 ### Project Management
-@claude-guidelines/project-management/build.md
-@claude-guidelines/project-management/testing.md
-@claude-guidelines/project-management/documentation.md
+@./claude-guidelines/project-management/build.md
+@./claude-guidelines/project-management/testing.md
+@./claude-guidelines/project-management/documentation.md
 
 ### Security & Operations
-@claude-guidelines/security.md
-@claude-guidelines/operations/monitoring.md
+@./claude-guidelines/security.md
+@./claude-guidelines/operations/monitoring.md
 
 ### API & Architecture
-@claude-guidelines/api-architecture/api-design.md
-@claude-guidelines/api-architecture/logging.md
-@claude-guidelines/api-architecture/observability.md
-@claude-guidelines/api-architecture/architecture.md
+@./claude-guidelines/api-architecture/api-design.md
+@./claude-guidelines/api-architecture/logging.md
+@./claude-guidelines/api-architecture/observability.md
+@./claude-guidelines/api-architecture/architecture.md
 
 ## Module Loading
 
 Auto-loaded via conditional loading based on task keywords and file types.
-@claude-guidelines/conditional-loading.md
+@./claude-guidelines/conditional-loading.md
 
 **NOTE**: Manual override available: `@load: security, performance` | `@skip: documentation` | `@focus: memory`
 

--- a/scripts/verify.sh
+++ b/scripts/verify.sh
@@ -89,6 +89,30 @@ check_executable() {
     fi
 }
 
+# Import 문법 검증 함수
+# Claude Code의 @import 문법은 상대 경로에 ./ 접두사가 필요합니다.
+# 올바른 예: @./path/to/file.md, @~/path/to/file.md, @/absolute/path.md
+# 잘못된 예: @path/to/file.md (./가 없음)
+validate_import_syntax() {
+    local file="$1"
+    local invalid_imports
+
+    # @로 시작하되 ./, ~/, /, http로 시작하지 않는 import 찾기
+    # 단, 예제 구문(@load:, @skip:, @focus:, @context:)과 Python 데코레이터(@app., @pytest. 등)는 제외
+    invalid_imports=$(grep -n '^@[^./~@]' "$file" 2>/dev/null | grep -v '^@https' | grep -v '^@load:' | grep -v '^@skip:' | grep -v '^@focus:' | grep -v '^@context:' | grep -v '@app\.' | grep -v '@pytest\.' | grep -v '@limiter\.' | grep -v '@before_' | grep -v '@after_' || true)
+
+    if [ -n "$invalid_imports" ]; then
+        error "Invalid import syntax in $file"
+        echo "  Use @./path for relative or @~/path for home directory"
+        echo "  Found:"
+        echo "$invalid_imports" | while read -r line; do
+            echo "    $line"
+        done
+        return 1
+    fi
+    return 0
+}
+
 # 백업 디렉토리 구조 검증
 echo ""
 echo "======================================================"
@@ -208,6 +232,62 @@ echo ""
 check_file "$BACKUP_DIR/README.md" "README.md"
 check_file "$BACKUP_DIR/QUICKSTART.md" "QUICKSTART.md"
 check_file "$BACKUP_DIR/HOOKS.md" "HOOKS.md (Hook 가이드)"
+
+# Import 문법 검증
+echo ""
+echo "======================================================"
+info "Import 문법 검증 (@import syntax)"
+echo "======================================================"
+echo ""
+
+# CLAUDE.md 파일들의 import 문법 검증
+IMPORT_CHECK_FILES=(
+    "$BACKUP_DIR/global/CLAUDE.md"
+    "$BACKUP_DIR/project/CLAUDE.md"
+)
+
+for check_file in "${IMPORT_CHECK_FILES[@]}"; do
+    if [ -f "$check_file" ]; then
+        TOTAL_CHECKS=$((TOTAL_CHECKS + 1))
+        if validate_import_syntax "$check_file"; then
+            success "$(basename "$(dirname "$check_file")")/$(basename "$check_file") import 문법 검증 통과"
+            PASSED_CHECKS=$((PASSED_CHECKS + 1))
+        else
+            FAILED_CHECKS=$((FAILED_CHECKS + 1))
+        fi
+    fi
+done
+
+# SKILL.md 파일들의 import 문법 검증
+if [ -d "$BACKUP_DIR/project/.claude/skills" ]; then
+    for skill_file in "$BACKUP_DIR/project/.claude/skills"/*/SKILL.md; do
+        if [ -f "$skill_file" ]; then
+            skill_name=$(basename "$(dirname "$skill_file")")
+            TOTAL_CHECKS=$((TOTAL_CHECKS + 1))
+            if validate_import_syntax "$skill_file"; then
+                success "skills/${skill_name}/SKILL.md import 문법 검증 통과"
+                PASSED_CHECKS=$((PASSED_CHECKS + 1))
+            else
+                FAILED_CHECKS=$((FAILED_CHECKS + 1))
+            fi
+        fi
+    done
+fi
+
+if [ -d "$BACKUP_DIR/plugin/skills" ]; then
+    for skill_file in "$BACKUP_DIR/plugin/skills"/*/SKILL.md; do
+        if [ -f "$skill_file" ]; then
+            skill_name=$(basename "$(dirname "$skill_file")")
+            TOTAL_CHECKS=$((TOTAL_CHECKS + 1))
+            if validate_import_syntax "$skill_file"; then
+                success "plugin/skills/${skill_name}/SKILL.md import 문법 검증 통과"
+                PASSED_CHECKS=$((PASSED_CHECKS + 1))
+            else
+                FAILED_CHECKS=$((FAILED_CHECKS + 1))
+            fi
+        fi
+    done
+fi
 
 # 통계
 echo ""

